test/e2e/Dockerfiile

```diff
@@ -0,0 +1,8 @@
+FROM registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.22-openshift-4.17 as builder
+WORKDIR /go/src/github.com/openshift/configuration-anomaly-detection
+COPY . .
+RUN  CGO_ENABLED=0 GOFLAGS="-mod=mod" go test ./test/e2e -v -c --tags=osde2e -o /harness.test
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+COPY --from=builder ./harness.test harness.test
+ENTRYPOINT [ "/harness.test" ]
```

Comment:
Copy this one from the latest https://github.com/openshift/configure-alertmanager-operator/blob/master/test/e2e/Dockerfile#L2-L9 we renamed `harness` just to `e2e`